app/assets/javascripts/application.js

```diff
@@ -14,4 +14,4 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require bootstrap-sprockets
-//= require_tree .
```

Comment:
I've already known it.. I've just wanted to pape specific javascript. If there is required_tree, all javascripts are loaded.
Thanks